From dwalker@fifo99.com  Fri Jul 10 15:30:14 2009
From: Daniel Walker <dwalker@fifo99.com>
Date: Sun,  5 Jul 2009 09:32:45 -0700
Subject: Staging: android: binder: partial checkpatch cleanup
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: Arve Hjønnevåg <arve@android.com>, Brian Swetland <swetland@google.com>, linux-kernel@vger.kernel.org, Daniel Walker <dwalker@fifo99.com>
Message-ID: <1246811565-8548-4-git-send-email-dwalker@fifo99.com>


Clean up 3 or so checkpatch errors from one of my prior patches.

Signed-off-by: Daniel Walker <dwalker@fifo99.com>
Cc: Arve Hjønnevåg <arve@android.com>
Cc: Brian Swetland <swetland@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/android/binder.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/staging/android/binder.c
+++ b/drivers/staging/android/binder.c
@@ -785,8 +785,8 @@ static struct binder_buffer *binder_allo
 	if (is_async) {
 		proc->free_async_space -= size + sizeof(struct binder_buffer);
 		binder_debug(BINDER_DEBUG_BUFFER_ALLOC_ASYNC,
-		             "binder: %d: binder_alloc_buf size %zd "
-		             "async free %zd\n", proc->pid, size,
+			     "binder: %d: binder_alloc_buf size %zd "
+			     "async free %zd\n", proc->pid, size,
 			     proc->free_async_space);
 	}
 
@@ -1146,7 +1146,7 @@ static void binder_delete_ref(struct bin
 		binder_debug(BINDER_DEBUG_DEAD_BINDER,
 			     "binder: %d delete ref %d desc %d "
 			     "has death notification\n", ref->proc->pid,
-		             ref->debug_id, ref->desc);
+			     ref->debug_id, ref->desc);
 		list_del(&ref->death->work.entry);
 		kfree(ref->death);
 		binder_stats_deleted(BINDER_STAT_DEATH);
@@ -2331,7 +2331,7 @@ retry:
 				if (!weak && !strong) {
 					binder_debug(BINDER_DEBUG_INTERNAL_REFS,
 						     "binder: %d:%d node %d u%p c%p deleted\n",
-					    	     proc->pid, thread->pid, node->debug_id,
+						     proc->pid, thread->pid, node->debug_id,
 						     node->ptr, node->cookie);
 					rb_erase(&node->rb_node, &proc->nodes);
 					kfree(node);
